go-controller/pkg/ovn/ovn.go

```diff
@@ -357,15 +340,6 @@ func (oc *Controller) ovnControllerEventChecker() {
 					klog.Errorf("Unable to remove controller event %s", event.uuid)
 					continue
 				}
-				if serviceName, ok := oc.GetServiceVIPToName(event.vip, event.protocol); ok {
-					serviceRef := kapi.ObjectReference{
-						Kind:      "Service",
-						Namespace: serviceName.Namespace,
-						Name:      serviceName.Name,
-					}
-					klog.V(5).Infof("Sending a NeedPods event for service %s in namespace %s.", serviceName.Name, serviceName.Namespace)
-					oc.recorder.Eventf(&serviceRef, kapi.EventTypeNormal, "NeedPods", "The service %s needs pods", serviceName.Name)
```

Comment:
This is needed for unidling to work; ovn-kubernetes sends the event to indicate that there is an incoming connection on the idled service, and [the unidling controller](https://github.com/openshift/openshift-controller-manager/tree/master/pkg/unidling/controller) sees the event and restarts the service's pods.

Reply:
yeah we do need that.